addons/web/static/src/scss/bootstrap_overridden.scss

```diff
@@ -294,8 +294,16 @@ $badge-color: inherit !default;
 $badge-padding-y: 0.25em !default;
 $badge-padding-x: 0.82em !default;
 
+<<<<<<< HEAD
```

Comment:
@mart-e - I don't believe I have a way of fixing this merge conflict, but this should point to `$gray-600` - also, is it supposed to go to the 16.0 branch or `master`? I had my PR based to 16.0

Reply:
@thatnerdjosh no the forward port branches are on our repository, we will adapt it :)